VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/dao/PatientVisitHistoryDao.java

```diff
@@ -19,7 +19,7 @@ public PatientVisitHistoryDao() throws Exception {
     }
 
     private static final String FIND_HISTORY_BY_PATIENT_ID_SQL =
-            "SELECT * FROM patient_visit_histories where patient_id = ?";
+            "SELECT * FROM patient_visit_histories where patient_id = ? order by id desc, appointment_date desc";
```

Comment:
If u order by Id first, since it is never duplicated, the second order by will not be triggered. I think it should be the other way around.

Reply:
I actually thought about this one, since id are inserted monotonically just ordering by ids is sufficient.... I think. Maybe I could just get rid of by appointment_date.